File: app/src/main/java/com/kota/Bahamut/Command/BahamutCommandLoadArticleEnd.java
package com.kota.Bahamut.Command;

import com.kota.Telnet.Reference.TelnetKeyboard;
import com.kota.Telnet.TelnetClient;

public class BahamutCommandLoadArticleEnd {
    public void execute() {
        System.out.print(toString() + " ");
        TelnetClient.getClient().sendKeyboardInputToServer(TelnetKeyboard.LEFT_ARROW);
    }

    public String toString() {
        return "[LoadArticleEnd]";
    }
}

File: app/src/main/java/com/kota/Bahamut/Command/BahamutCommandLoadArticleEnd.kt
package com.kota.Bahamut.Command

import com.kota.Telnet.Reference.TelnetKeyboard
import com.kota.Telnet.TelnetClient

class BahamutCommandLoadArticleEnd {
    fun execute() {
        print(toString() + " ")
        TelnetClient.getClient().sendKeyboardInputToServer(TelnetKeyboard.LEFT_ARROW)
    }

    override fun toString(): String {
        return "[LoadArticleEnd]"
    }
}
